bedrock/firefox/templates/firefox/hub/home.html

```diff
@@ -131,10 +131,10 @@ <h2 class="card-heading">{{ _('Take Firefox to go!') }}</h2>
               </a>
 
               <p>
-              {% trans url1=url('firefox.ios'), url2=url('firefox.android.index') %}
+              {% trans url=url('firefox.mobile') %}
                 Get your favorite bookmarks, passwords & more on any
-                <a href="{{ url1 }}">iOS</a> and
-                <a href="{{ url2 }}">Android</a> device with Firefox Sync.
+                <a href="{{ url }}">iOS</a> and
```

Comment:
This is going to break translations I think. You might need to keep the same variable names to avoid altering the string?

Reply:
I was wondering if this would result in a string change. I'll go back to two variables here.